nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -225,6 +214,26 @@ let
         set menu_color_highlight=white/blue
       fi
     ''}
+
+    set loadfontdone=false
+    $(find ${config.isoImage.grubTheme} -iname '*.pf2' -printf "if loadfont (\$root)/EFI/BOOT/grub-theme/%P; then set loadfontdone=true; fi\n")
```

Comment:
Font loading occurs a few lines higher up in this file. Shouldn't that be changed instead of adding this?